pandas_dash_app.py

```diff
@@ -0,0 +1,60 @@
+import dash
+import dash_table
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output
+import pandas as pd
+
+### Generate the (pandas) data
+test_list = ['a', 'b', 'c', 'd']
+pdf = pd.DataFrame(test_list, columns=['letters'])
+
+### Some useful functions 
+
+# Generate a more basic html table UI output
```

Comment:
Do you wanna remove unused code perhaps? It is in the commit history so unnecessary ;)